nx.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "nx/presets/npm.json",
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "nxCloudAccessToken": "NmU2OTYyZTgtYWQ2Zi00NzNkLWIzNDQtZTlhZjEwZDI0OTcxfHJlYWQtd3JpdGU="
```

Comment:
Oh sorry can we take out this stuff related to nx cloud please 🙏

Doesn't look like it's being used but we wouldn't want our code getting forwarded out to them for processing so probably safest to remove reference to it.